doc/whats-new.rst

```diff
@@ -21,6 +21,12 @@ v2024.09.1 (unreleased)
 
 New Features
 ~~~~~~~~~~~~
+- ``DataTree`` related functionality is now exposed in the main ``xarray`` public
+  API. This includes: ``xarray.DataTree``, ``xarray.open_datatree``,
+  ``xarray.map_over_subtree``, ``xarray.register_datatree_accessor`` and
+  ``xarray.testing.assert_isomorphic``.
+  By `Owen Littlejohns <https://github.com/owenlittlejohns>`_ and
+  `Tom Nicholas <https://github.com/TomNicholas>`_.
```

Comment:
This would be a good place to try to list everyone who contributed to the DataTree launch, e.g., at least also yourself, me and Eni :)

Reply:
Also Owen and Justus!